build.gradle

```diff
@@ -315,8 +315,7 @@ task fullAcceptanceTest(type: GradleBuild) {
 task acceptanceTest(type: Test) {
     testClassesDir = sourceSets.acceptanceTest.output.classesDir
     classpath = sourceSets.acceptanceTest.runtimeClasspath
-//    maxParallelForks = Runtime.runtime.availableProcessors()
-    maxParallelForks = 1
+    maxParallelForks = Runtime.runtime.availableProcessors()
```

Comment:
@antban Why have you changed it now? I thought you told that it's not stable yet to run tests in parallel.